pysteps/extrapolation/semilagrangian.py

```diff
@@ -140,10 +152,10 @@ def interpolate_motion(D, V_inc, td):
         XYW = [XYW[1, :, :], XYW[0, :, :]]
 
         VWX = ip.map_coordinates(
-            velocity[0, :, :], XYW, mode="nearest", order=0, prefilter=False
+            velocity[0, :, :], XYW, mode="nearest", order=1, prefilter=False,
```

Comment:
can you briefly explain why the interpolation order here needs to be 1?

Reply:
Since we are integrating along the advection field, using the nearest interpolation might not be a good idea because the errors accumulate over lead time. I noticed that setting order=0 caused artefacts in some cases but they disappeared by setting order=1.